cves/kernel/CVE-2019-20636.yml

```diff
@@ -151,10 +144,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: "The individual who messaged about the vulnerability was Dmitry Torokhov,
+    an individual who was a senior software engineer at Google at the time of the
+    discovery."
```

Comment:
I'd argue this is more about who discovered the vulnerability, not how it was discovered. I could be misinterpreting it, though.